qiskit-superstaq/qiskit_superstaq/compiler_output.py

```diff
@@ -59,28 +59,33 @@ def measured_qubit_indices(circuit: qiskit.QuantumCircuit) -> list[int]:
     return sorted(circuit.find_bit(qubit).index for qubit in measured_qubits)
 
 
-def measured_clbit_indices(circuit: qiskit.QuantumCircuit) -> list[int]:
-    """Returns the indices of the classical bits in the input quantum circuit.
+def classical_bit_mapping(circuit: qiskit.QuantumCircuit) -> dict[int, int]:
+    """Returns the index of the (final) measured qubit associated with each classical bit.
+
+    If more than one measurements are assigned to the same classical bit, only the final measurement
+    is considered.
```

Comment:
micro-nit: may sound more natural, but could just be wording preference
```suggestion
    If more than one measurement is assigned to the same classical bit, only the final measurement
    is considered.
```

Reply:
agreed, thanks